### PR TITLE
Feature/view profile

### PR DESCRIPTION
**Pull request by rinkishivani**

### CODE REVIEW

File: DashClothes/settings.py [hunk 1]
```diff
@@ -129,3 +129,11 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 AUTH_USER_MODEL = "accounts.User"
+
+ACCOUNT_EMAIL_REQUIRED = True
+ACCOUNT_EMAIL_VERIFICATION = 'mandatory'
+ACCOUNT_AUTHENTICATED_LOGIN_REDIRECTS = True
+LOGIN_REDIRECT_URL = 'homepage'
+# LOGOUT_REDIRECT_URL = 'account_login'
```
Comment (palakparmartntra) [inline DashClothes/settings.py]: remove unnecessary commented code.

File: accounts/views.py [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+    """ To view user details """
+    model = User
+    template_name = 'profile/view_profile.html'
+    context_object_name = 'user'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        user = self.get_object()
+        context['addresses'] = user.address.all()
+        context['other_address_displayed'] = False
```
Comment (palakparmartntra) [inline accounts/views.py]: what is the purpose of context['other_address_displayed' ?  @rinkishivani

File: templates/profile/view_profile.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                    </div>
+                </div>
+                    <h2>Address</h2><br>
+                    {% for address in addresses %}
+                    <div class="row">
+                    <div class="col-md-6">
+                            <h3>
+                                {% if address.is_primary %}
+                                    Default Address
+                                {% elif not other_address_displayed %}
```
Comment (palakparmartntra) [inline templates/profile/view_profile.html]: not getting this condition @rinkishivani

File: DashClothes/settings.py (hunk 1 above)
Comment (rinkishivani) [reply DashClothes/settings.py]: removed

File: accounts/views.py (hunk 2 above)
Comment (rinkishivani) [reply accounts/views.py]: removed other address displayed.

File: templates/profile/view_profile.html (hunk 3 above)
Comment (rinkishivani) [reply templates/profile/view_profile.html]: changed elif to else

**Pull request closed without merging**